sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py

```diff
@@ -323,64 +603,49 @@ def test_create_service_with_connection_string_key(self):
 
     def test_create_service_with_connection_string_sas(self):
         # Arrange
-        self.sas_token = self.generate_sas_token()
-        sas_token = AzureSasCredential(self.sas_token)
-        conn_string = "AccountName={};SharedAccessSignature={};".format(
-            self.tables_cosmos_account_name, sas_token.signature
-        )
-        endpoint_suffix = os.getenv("TABLES_COSMOS_ENDPOINT_SUFFIX", DEFAULT_COSMOS_ENDPOINT_SUFFIX)
+        sas_token = self.generate_sas_token()
+        conn_string = f"AccountName={self.tables_cosmos_account_name};SharedAccessSignature={AzureSasCredential(sas_token).signature};TableEndpoint=www.mydomain.com"
 
-        for client, url in SERVICES.items():
+        for client in SERVICES:
             # Act
-            service = client.from_connection_string(conn_string, table_name="foo", endpoint_type=url)
+            service = client.from_connection_string(conn_string, table_name="foo")
 
             # Assert
             assert service is not None
-            assert service.url.startswith("https://" + self.tables_cosmos_account_name + ".table." + endpoint_suffix)
+            assert service.url.startswith("https://www.mydomain.com")
```

Comment:
Why did this change?

Reply:
I reconstructed the conn_string with a fake endpoint `www.mydomain.com`.